pkg/arvo/sys/zuse.hoon

```diff
@@ -5070,7 +5070,7 @@
   |=  ord=$-([key key] ?)
   |=  a=*
   =/  b  ;;((tree [key=key val=value]) a)
-  ?>  (check-balance:((ordered-map key value) ord) b)
+  ?>  (apt:((ordered-map key value) ord) b)
   b
 ::
 ::  $mk-item: constructor for +ordered-map item type
```

Comment:
While you're cleaning things up generally, I don't see any reason for `+mk-item` to be a top-level name. I'd just replace `$item` below:

```
+$   item  [=key =val]
```